worker/jwtparser/worker_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package jwtparser_test
```

Comment:
We don't do this convention anymore sorry. We can safely just put this under the jwtparser package and remove the export in export_test.go :)